pom.xml

```diff
@@ -16,28 +16,30 @@
   </licenses>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
 
     <logback-version>1.1.2</logback-version>
     <scala-maven-plugin-version>3.2.1</scala-maven-plugin-version>
-    <sbv>2.11</sbv>
-    <scala-version>${sbv}.7</scala-version>
+    <sbv>3.3</sbv>
+    <scala-version>${sbv}.0</scala-version>
```

Comment:
You can clean the following dependencies too, it's better to start as clean as possible and only introduce again what is really needed with the new stack. Esp. thinks like joda-time, it's sure we will want to use `java.date`